reporters/cloud-run-scripts/CloudResponse.js

```diff
@@ -1,24 +1,48 @@
 const express = require("express");
 const app = express();
+app.use(express.json()); // Enables json to be parsed by the express process.
 const { exec } = require("child_process");
 
-app.get("/", (req, res) => {
-  console.log("Daily reporter received a request");
+app.post("/", (req, res) => {
+  console.log("Executing GCP Cloud Run API");
+  if (!req.body.cloudRunCommand) {
+    res.status(400).send({
+      message: "ERROR: Body missing json Cloud Run Command!"
+    });
+    throw "Missing cloudRunCommand";
+  }
 
-  let reporterScript = exec("/bin/bash ../reporters/cloud-run-scripts/SendSlackReport.sh", (error, stdout, stderr) => {
-    console.log(stdout);
-    console.log(stderr);
+  // Iterate over the provided environment variables and ensure that they are all strings. This enables json configs
+  // to be passed in the req body and then set as environment variables in the child_process as a string
+  let processedEnvironmentVariables = {};
+  Object.keys(req.body.environmentVariables).forEach(key => {
```

Comment:
nit: what happens if `environmentVariables` doesn't exist in the body?

Reply:
That's a good point. it should be enforced in the forEach I have updated accordingly.